cdk/apps/rnasum/app.py

```diff
@@ -2,7 +2,36 @@
 from aws_cdk import core
 from stacks.iap_tes import IapTesStack
 from stacks.cicd import CICDStack
+from stacks.batch import BatchStack
+from stacks.common import CommonStack
 
+rnasum_image = 'umccr/rnasum:0.3.5'
+
+dev_env = {'account': '843407916570', 'region': 'ap-southeast-2'}
+codebuild_project_name = 'rnasum_codebuild_project'
+rnasum_ecr_repo = 'rnasum'
+
+common_dev_props = {
+    'namespace': 'rnasum-common-dev',
+    'rnasum_ecr_repo': rnasum_ecr_repo,
+    'vpc_id': 'vpc-0c459c431bac7d497'       # reuse umccrise common vpc for now
```

Comment:
As discussed, we might want to delegate VPCs to terraform... unsure about this dual-deploy-stack experiment (TF for infra, CDK for apps), we'll see how it pans out.

Reply:
What are your concerns? The disconnection?
I guess we'll always have that... 

I thought about storing critical information that is used in multiple stacks/setups in a central place. E.g. Terraform could push VPC IDs to AWS parameter store, where it could be picked up by CDK stacks automatically,... but perhaps a bit overkill at this point (and it comes with it's own side effects).

I am happy to continue this path and see how it pans out...

Reply:
Yap; not really intent to use `umccrise` VPC here. When common VPC for apps (boostrap?) is ready, will update this VPC ID.